params/protocol_params.go

```diff
@@ -69,6 +69,32 @@ const (
 	MemoryGas        uint64 = 3     // Times the address of the (highest referenced byte in memory + 1). NOTE: referencing happens on read, write and in instructions such as RETURN and CALL.
 	TxDataNonZeroGas uint64 = 68    // Per byte of data attached to a transaction that is not equal to zero. NOTE: Not payable on data of calls between transactions.
 
+	// These have been changed during the course of the chain
+	CallGasFrontier        uint64 = 40  // Once per CALL operation & message call transaction.
+	CallGasEip150          uint64 = 700 // Static portion of gas for CALL-derivates after EIP 150 (T.W)
+	BalanceGasFrontier     uint64 = 20  // The cost of a BALANCE operation
+	BalanceGasEip150       uint64 = 400 // The cost of a BALANCE operation after Tangerine Whistle
+	ExtcodeSizeGasFrontier uint64 = 20  // Cost of EXTCODESIZE before EIP 150 (T.W)
```

Comment:
Nitpick, please refrain from the `T.W` abbreviation, it's not a standard. Let's use `Tangerine` if we don't want to write it all out.

Reply:
I like to write both eip number and name.. I often become confused as to which is which unless it's written out explicitly. I'll change to Tangerine